requirements-test.txt

```diff
@@ -12,7 +12,7 @@ Automat==0.7.0
 Babel==2.7.0
 beautifulsoup4==4.8.0
 billiard==3.6.1.0
-black==19.3b0
+black==19.10b0
```

Comment:
I'm aiming to make this file redundant. It's best off including this in the `test` requirements in the setup.py :-)

Reply:
Yep, I'd like to retire it too. A bump is needed in the short-term to fix the current issue: pre-commit is installing the latest Black, but this `requirements-test.txt` is pinning an older Black.

setup.py does has Black in the test requirements already, but doesn't specify anything about versions. Do you think we should?